lib/perso.ml

```diff
@@ -1216,6 +1239,295 @@ let warning_use_has_parents_before_parent (fname, bp, ep) var r =
   |> !GWPARAM.syslog `LOG_WARNING;
   r
 
+let cousins_t = ref None
+let cousins_dates_t = ref None
+
+let max_l1_l2_aux env =
+  let max_l1 = match get_env "max_anc_level" env with Vint i -> i | _ -> 5 in
+  let max_l2 = match get_env "max_desc_level" env with Vint i -> i | _ -> 5 in
```

Comment:
what does those 5 correspond to? maybe you want to define a global default value